src/components/shared/FilterNavigatorBar.js

```diff
@@ -81,18 +102,16 @@ export class FilterNavigatorBar extends React.PureComponent<Props> {
             classNames="filterNavigatorBarUncommittedTransition"
             timeout={0}
           >
-            <li
-              className={classNames(
-                'filterNavigatorBarItem',
-                'filterNavigatorBarLeafItem',
-                'filterNavigatorBarUncommittedItem'
-              )}
+            <FilterNavigatorBarListItem
+              index={items.length}
+              isFirstItem={false}
+              isLastItem={true}
+              isSelectedItem={false}
+              additionalClassName="filterNavigatorBarUncommittedItem"
               title={uncommittedItem}
```

Comment:
Huh, interesting that we are putting a title only for uncommited items, and this is exactly the same text we put inside the button as well 🤔 I feel like this is redundant (but not a blocker for this PR since this was a preexisting issue).